pennylane/beta/devices/default_tensor_tf.py

```diff
@@ -54,50 +53,25 @@ class DefaultTensorTF(DefaultTensor):
 
     **Example**
 
-    The ``default.tensor.tf`` device supports various differentiation modes.
+    The ``default.tensor.tf`` device supports end-to-end classical backpropagation with the TensorFlow interface.
 
-    * *End-to-end classical backpropagation with the TensorFlow interface*.
-      Using this method, the created QNode is a 'white-box', and is
-      tightly integrated with your TensorFlow computation:
+    Using this method, the created QNode is a 'white-box', and is
+    tightly integrated with your TensorFlow computation:
 
-      >>> dev = qml.device("default.tensor.tf", wires=1)
-      >>> @qml.qnode(dev, interface="tf", diff_method="backprop")
-      >>> def circuit(x):
-      ...     qml.RX(x[1], wires=0)
-      ...     qml.Rot(x[0], x[1], x[2], wires=0)
-      ...     return qml.expval(qml.PauliZ(0))
-      >>> vars = tf.Variable([0.2, 0.5, 0.1])
-      >>> with tf.GradientTape() as tape:
-      ...     res = circuit(vars)
-      >>> tape.gradient(res, vars)
-      <tf.Tensor: shape=(3,), dtype=float32, numpy=array([-2.2526717e-01, -1.0086454e+00,  1.3877788e-17], dtype=float32)>
+    >>> dev = qml.device("default.tensor.tf", wires=1)
+    >>> @qml.qnode(dev, interface="tf", diff_method="backprop")
+    >>> def circuit(x):
+    ...     qml.RX(x[1], wires=0)
+    ...     qml.Rot(x[0], x[1], x[2], wires=0)
+    ...     return qml.expval(qml.PauliZ(0))
+    >>> vars = tf.Variable([0.2, 0.5, 0.1])
+    >>> with tf.GradientTape() as tape:
+    ...     res = circuit(vars)
+    >>> tape.gradient(res, vars)
+    <tf.Tensor: shape=(3,), dtype=float32, numpy=array([-2.2526717e-01, -1.0086454e+00,  1.3877788e-17], dtype=float32)>
 
-      In this mode, you must use the ``"tf"`` interface, as TensorFlow
-      is used as the device backend.
-
-    * *Device differentiation*. Using this method, the created QNode
```

Comment:
I removed device differentiation from the `default.tensor.tf` device, since it relies on the old Variable class (and it is superseded by adjoint diff)